src/shared/constants/index.ts

```diff
@@ -22,7 +22,7 @@ export const Constants = {
   DEFAULT_NODEPOOL_NAME: 'Unnamed node pool',
   AZURE_NODEPOOL_DEFAULT_VM_SIZE: 'Standard_D4_v3',
   AWS_V5_VERSION: '10.0.0',
-  AZURE_V5_VERSION: '40.0.0',
+  AZURE_V5_VERSION: '13.0.0-alpha',
```

Comment:
While we're here, also change the default Azure v5 version to the one that we'll use in production (the version here only gets used if it is not present in the API response.